examples/panel/book-recommender/rag_book_recommender.py

```diff
@@ -0,0 +1,89 @@
+import os
+import json
+import pickle
+import pandas as pd
+
+from pathlib import Path
+from openai import OpenAI
+
+client = OpenAI()
+
+file_path = os.path.join('assets', 'goodreads.csv')
+df = pd.read_csv(file_path)
+
+
+file_path = os.path.join('assets', 'title_to_description.pkl')
+with open(file_path, 'rb') as file:
+    DESCRIPTIONS = pickle.load(file)
+
+
+def get_authors():
+    file_path = os.path.join('assets', 'author_to_title.pkl')
+    with open(file_path, 'rb') as file:
+        authors = pickle.load(file)
+    return authors
+
+
+def get_embeddings():
+    file_path = os.path.join('assets', 'embeddings.json')
+    with open(file_path, "r", encoding="utf-8") as file:
+        embeddings_json = json.load(file)
+    return embeddings_json
+
+
+def get_book_description_by_title(title):
+    return DESCRIPTIONS[title.upper()]
+
+
+class EmbeddingsStore:
+    def __init__(self):
+        self._path = Path("embeddings_another.json")
```

Comment:
what's the difference between embeddings.json and embeddings_another.json?

Reply:
Added ` embeddings_another.json` by mistake